model/fv_control.F90

```diff
@@ -141,6 +141,7 @@ module fv_control_mod
    use fv_mp_mod,           only: mp_start, domain_decomp, mp_assign_gid, global_nest_domain
    use fv_mp_mod,           only: broadcast_domains, mp_barrier, is_master, setup_master, grids_master_procs, tile_fine
    use fv_mp_mod,           only: MAX_NNEST, MAX_NTILE
+   use fv_processmodel_mod, only: read_namelist_fv_processmodel
```

Comment:
In fv_restart.F90, you protected a similar module use statement with a compile-time conditional.  Should you do the same here?  Are you advocating the process model code always be compiled, or conditionally compiled?

Reply:
Good question. Since it would only be used in UFS, it would make sense to exclude it from non-UFS builds, so that would be consistent with fv_restart.F90. I'll add the GFS_PHYS flag here, too -- must have just been missed. Thanks!